ui/DesignSystem/Component/Topbar/Menu.svelte

```diff
@@ -0,0 +1,48 @@
+<script>
+  import { location } from "svelte-spa-router";
+  import * as path from "../../../lib/path.js";
+
+  import MenuItem from "./MenuItem.svelte";
+
+  export let items = [];
```

Comment:
This was something I was curious about - if this is `null`, the `#each` block will throw an error (`null` has no `length`) if the prop isn't received for some reason. If it's an empty array, it'll at least render without error. Do you think that's being too explicit about the type?

Reply:
I see what you mean. What I'm wondering is, shouldn't it be an exception in case one forgets a required prop as a reminder?